### ISSUE TITLE
Profile: Page features, logic updates, etc. 

### ISSUE THREAD

**Issue by khoadnguyen** [SoftStackFactory/reboot#298]
**Describe the feature briefly:**
The profile page could use some clean up to actually fit the requirements. Please refer to the project charter if you need more information. 

**What needs to be done? (please complete in this order)**
- [x] Update Profile page inputs styling/layout #316
- [x] Update Profile page input cohesion/logic to match wizard section #317
- [x] Add input validation and api calls to backend to update user info #318

**Where does this appear?**
Profile Page

**Extra notes:**
Depends on #318, #317, #316

**Comment by KevinChheang** [comment]
I see you combined every tasks into one issue. Is it ok if I break it up individually but use the same issue number?

For example: "298-profile-image-alignment"
                       "298-profile-api-call" and so on

**Comment by khoadnguyen** [comment]
@KevinChheang Yea, break them up for you. Do you want each on its own issue?

**Comment by KevinChheang** [comment]
Isn’t what we have been doing is to keep every commit small so it’s easy to debug. But if you want to keep it this way it’s totally fine with me. I was just curious that’s all.

**Comment by khoadnguyen** [comment]
Well it dawned on me that the Updating the logic and the api call updates are dependent on each other. The styling possibly could be affected by it as well, so then I just put them all together. 

Generally, yes if it makes sense you should try to keep them apart as much as possible. However sometimes you can't break two items apart without making it harder to develop on. If you'd prefer one way, I can break it out of I can keep them together.

**Comment by khoadnguyen** [comment]
Reassigning to @AlexRingrose
